src/modules/I18n/i18next-scanner.config.js

```diff
@@ -4,13 +4,25 @@ const {
 const fg = require('fast-glob');
 const typescriptTransform = require('i18next-scanner-typescript');
 
-const namespaces = fg
+const moduleNamespaces = fg
   .sync(['src/modules/*', 'src/modulesAdmin/*', 'src/api/modules/*'], {
     onlyDirectories: true,
     deep: 1,
   })
   .filter((o) => !o.includes('__tests__'))
-  .map((o) => o.toLowerCase().replace('src/modules/', ''));
+  .map((o) => o.toLowerCase().replace('src/modules/', 'modules/'));
+
+const pagesNamespaces = fg
+  .sync(['src/pages/**/*.tsx'], { onlyFiles: true, ignore: ['src/pages/_app.tsx'] })
+  .map((o) => o.toLowerCase().replace('src/pages/', ''))
```

Comment:
Not sure what this code does and why it chanegd so much compared to before, can you please elaborate

(especially why we did not have to replace `tsx` by `` before

Reply:
This code build the namespaces for every `.tsx`file in `src/pages/`, it allows to write one translation file in `src/translations/${locale}/${pagename}.json`

We did not have to replace `tsx`by `` before  because we don't make a namespace per page.